src/difference/solution-name.spec.js

```diff
@@ -0,0 +1,43 @@
+import {
+  difference1,
+  difference2,
+  difference3,
+  difference4,
+} from './alina/sandbox.js';
+
+for (const solution of [difference1, difference2, difference3, difference4]) {
+  describe('test difference of two arrays and without duplication', () => {
+    describe('default parameters', () => {
+      it('argument values is empty', () => {
+        const array = [];
+        const actual = solution(array);
+        expect(actual).toEqual([]);
+      });
+      it('both argument are empty', () => {
+        const actual = solution();
+        expect(actual).toEqual([]);
+      });
+    });
+
+    describe('standard test', () => {
+      it('number array', () => {
```

Comment:
it case tests need to include the word 'should'

example: should return number array

Reply:
`should`... got it.